roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -344,5 +370,212 @@
+    "os_compute_api:os-server-password:discoverable": "",
+    "os_compute_api:os-server-usage": "",
+    "os_compute_api:os-server-usage:discoverable": "",
+    "os_compute_api:os-server-groups": "",
+    "os_compute_api:os-server-groups:discoverable": "",
+    "os_compute_api:os-services": "rule:admin_api",
+    "os_compute_api:os-services:discoverable": "",
+    "os_compute_api:server-metadata:discoverable": "",
+    "os_compute_api:server-metadata:index": "rule:admin_or_owner",
+    "os_compute_api:server-metadata:show": "rule:admin_or_owner",
+    "os_compute_api:server-metadata:delete": "rule:admin_or_owner",
+    "os_compute_api:server-metadata:create": "rule:admin_or_owner",
+    "os_compute_api:server-metadata:update": "rule:admin_or_owner",
+    "os_compute_api:server-metadata:update_all": "rule:admin_or_owner",
+    "os_compute_api:servers:discoverable": "",
+    "os_compute_api:os-shelve:shelve": "",
+    "os_compute_api:os-shelve:shelve:discoverable": "",
+    "os_compute_api:os-shelve:shelve_offload": "rule:admin_api",
+    "os_compute_api:os-simple-tenant-usage:discoverable": "",
+    "os_compute_api:os-simple-tenant-usage:show": "rule:admin_or_owner",
+    "os_compute_api:os-simple-tenant-usage:list": "rule:admin_api",
+    "os_compute_api:os-suspend-server:discoverable": "",
+    "os_compute_api:os-suspend-server:suspend": "rule:admin_or_owner",
+    "os_compute_api:os-suspend-server:resume": "rule:admin_or_owner",
+    "os_compute_api:os-tenant-networks": "rule:admin_or_owner",
+    "os_compute_api:os-tenant-networks:discoverable": "",
+    "os_compute_api:os-shelve:unshelve": "",
+    "os_compute_api:os-user-data:discoverable": "",
+    "os_compute_api:os-virtual-interfaces": "",
+    "os_compute_api:os-virtual-interfaces:discoverable": "",
+    "os_compute_api:os-volumes": "",
+    "os_compute_api:os-volumes:discoverable": "",
+    "os_compute_api:os-volumes-attachments:index": "",
+    "os_compute_api:os-volumes-attachments:show": "",
+    "os_compute_api:os-volumes-attachments:create": "",
+    "os_compute_api:os-volumes-attachments:update": "",
+    "os_compute_api:os-volumes-attachments:delete": "",
+    "os_compute_api:os-volumes-attachments:discoverable": "",
+    "os_compute_api:os-availability-zone:list": "",
+    "os_compute_api:os-availability-zone:discoverable": "",
+    "os_compute_api:os-availability-zone:detail": "rule:admin_api",
+    "os_compute_api:os-used-limits": "rule:admin_api",
+    "os_compute_api:os-used-limits:discoverable": "",
+    "os_compute_api:os-migrations:index": "rule:admin_api",
+    "os_compute_api:os-migrations:discoverable": "",
+    "os_compute_api:os-assisted-volume-snapshots:create": "rule:admin_api",
+    "os_compute_api:os-assisted-volume-snapshots:delete": "rule:admin_api",
+    "os_compute_api:os-assisted-volume-snapshots:discoverable": "",
+    "os_compute_api:os-console-auth-tokens": "rule:admin_api",
+    "os_compute_api:os-server-external-events:create": "rule:admin_api"
```

Comment:
This is a whole lot of new policy, were these things just not covered before in upstream files, or...?